app/controllers/api/v1/uploads_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::UploadsController < ApplicationController
```

Comment:
This should eventually be changed to inherit from the `Api::V1::BaseController`. That'll handle the authentication. That can be done when you change the front end to use the new fetch implementation.